### PR TITLE
Log RANDOM_SEED (backport to 2.4)

### PR DESCRIPTION
**Pull request by jondequinor**
**Issue**
 Resolves equinor/ert#309

**Approach**
 Reinstate 82954b5#diff-57ee1e547e5d8ce7df030c6a42822abbL160

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by andreabrambilla** [comment]
jenkinst test this please

**Comment by andreabrambilla** [comment]
I don't fully understand what's the relation between this pr and #642 
Would it make sense to merge them into a single PR?

**Comment by andreabrambilla** [comment]
Nevermind, I noticed this is a backport :+1:

**Review by andreabrambilla — APPROVED** [review]
The fix looks good, just make sure it builds before merging 👍 

(jenkins won't manage to build this automatically cause it cannot detect what branch it is to be merged with, it defaults to master)

File: lib/enkf/rng_config.cpp [hunk 1]
```diff
@@ -181,6 +181,7 @@ void rng_config_init(rng_config_type * rng_config, const config_content_type * c
   if(config_content_has_item(config_content, RANDOM_SEED_KEY)) {
     const char * random_seed = config_content_get_value(config_content, RANDOM_SEED_KEY);
     rng_config_set_random_seed(rng_config, random_seed);
+    res_log_critical("Using RANDOM_SEED: %s", random_seed)
```
Comment (andreabrambilla) [inline lib/enkf/rng_config.cpp]: I guess this should be `fcritical`

**Comment by jondequinor** [comment]
Built on ubuntu:18.04.

**Pull request merged**